models/reviewsModel.js

```diff
@@ -53,3 +53,20 @@ exports.fetchReviewComments = (reviewId) => {
       throw err;
     });
 };
+
+exports.insertComment = (username, body, reviewId) => {
+  return db
+    .query(
+      `INSERT INTO comments (review_id, author, body)
+      VALUES ($1, $2, $3)
+      RETURNING *;
+      `,
+      [reviewId, username, body]
+    )
+    .then((results) => {
+      return results.rows[0];
+    })
+    .catch((err) => {
```

Comment:
ideally we should be catching errors in the controller and passing them to error handling middleware rather than in the model